Resources/public/js/views/services/ez-sectionserversideviewservice.js

```diff
@@ -39,6 +39,14 @@ YUI.add('ez-sectionserversideviewservice', function (Y) {
                 tasks = new Y.Parallel(),
                 contentService = this.get('capi').getContentService();
 
+            this.fire('notify', {
+                notification: {
+                    identifier: 'assign-section',
```

Comment:
I've just realized that the two notifiy events are fired in 2 different components. I think they should be both fired from the view service.